etc/fibo-vue/src/components/ModuleTree.vue

```diff
@@ -3,12 +3,6 @@
     <div class="arrow-container" :class="{hidden: !isFolder}" @click="toggle">
       <i :class="{down: isOpen}" />
     </div>
```

Comment:
Whoa, something amiss if this PR is changing the .vue website file(s)

Reply:
This file and the other one, below, were somehow pulled in when I merged from master, so I think someone else did something that caused this - hoping that it doesn't have a negative impact